src/compare-tree/index.js

```diff
@@ -1,28 +1,31 @@
+/**
+ * @flow
+ * @private
+ */
+
 import Node from './node'
 import Query from './query'
-import { getKeyString } from '../key-path'
+import { getKeyString, type KeyPath } from '../key-path'
```

Comment:
is this importing a class and then turning it into a flow type from `key-path`...if so...TIL. Is this different than just importing the class (without saying `type`) and just using that as a type?